fields/types/number/NumberValidation.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+
+	ValidateNumber: function (value, rules, field) {
```

Comment:
Uppercase function names should be reserved for classes, this would be better as `validateNumber`